internal/indexer/types/types.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 type Account struct {
-	StellarAddress string    `json:"stellarAddress,omitempty" db:"stellar_address"`
+	StellarAddress string    `json:"address,omitempty" db:"stellar_address"`
```

Comment:
Any reason not to change this to `address` within the struct rather than just the json representation?

Reply:
@JakeUrban We are reading the json tag and mapping it to db tags so that is what needs to be changed. However, I do want to change the struct variable to Address too but thats outside the scope of this PR. Will do it in a separate one